source/administration/object-management/object-versioning.rst

```diff
@@ -149,12 +147,12 @@ uses a ``null`` version ID. You can access or remove these objects by specifying
 Versioned Delete Operations
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Performing a ``DELETE`` operation on a versioned object creates a 
-0-byte ``DeleteMarker`` as the latest version of that object. Clients performing
-``GET`` operations on that object do not return any results, as MinIO does not
-return the ``DeleteMarker`` back as part of the response. Similarly, performing
-a ``LIST`` operation by default returns only objects which are *not* a
-``DeleteMarker``.
+Performing a ``DELETE`` operation on a versioned object creates a 0-byte ``DeleteMarker`` as the latest version of that object.
+For objects where the latest version is a ``DeleteMarker``, clients must specify versioning flags or identifiers to ``GET/HEAD/LIST/DELETE`` operations to work on any version of that object.
```

Comment:
Does this mean the client must request a specific version if the most recent is a delete marker?

I'm not sure if `to work on any version of that object` refers to any/all previous (not deleted) versions, or only a specific one.

Reply:
maybe "any previous version of that object" ?

Reply:
I think that makes sense 👍🏻